lib/house.rb

```diff
@@ -0,0 +1,31 @@
+class House
+  attr_reader :price, :address, :rooms
+
+  def initialize(price, address)
+    @price = price
+    @address = address
+    @rooms = []
+  end
+
+  def add_room(room)
+    @rooms << room
+  end
+
+  def above_market_average?
+    if @price.delete!("$").to_i > 500000
+      true
+    else
+      false
+    end
+  end
+
+  def rooms_from_category(category)
```

Comment:
Great job using each here to get the job done. `find_all` would save you a few lines here: 
```
@rooms.find_all do |room|
room.category == category
end
```